_source/_posts/2024-04-11-okta-powershell-module-podcast.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: blog_post
+title: "Introducing Okta's Official PowerShell Module"
+author: edunham
+by: advocate
+communities: [devops,.net]
+description: "In this podcast, we discuss the new PowerShell module with Laura Rodriguez and Bhavik Thakkar"
```

Comment:
```suggestion
description: "In this podcast, we discuss the new PowerShell module that enables you to manage your Okta resources with Laura Rodriguez and Bhavik Thakkar"
```

Recommending adding a "why is this something one would watch" in here since this is the description that shows as a short preview for the link (and within Google search results). But please feel free to change the text to whatever is appropriate